website/docs/apps/controller_app/README.md

```diff
@@ -0,0 +1,97 @@
+---
+id: controller-app
+title: Autoplot App
+---
+
+# Autoplot App Guide
+
+The Autoplot app is an advanced control interface for your Amiga, offering multiple modes of
+operation such as teach-and-repeat, WASD control, and remote access through a web browser.
+
+## How to Use the Autoplot App
+
+### Teach-and-Repeat
+
+The teach-and-repeat functionality allows you to manually drive the robot along a desired path
+and then have the robot repeat the path autonomously.
+
+1. Click "Record" to start mapping your path.
+2. Drive the robot as desired.
+3. Click "Save" and name your track to use it later.
+4. Load the track when needed and click "Start Track" to initiate autonomous follow.
+
+### Remote Access via Web Browser
+
+The Autoplot is a web-app, and can be access through any web browser, such as Google Chrome.
+To view it in your browser, navigate to: `<element-vegetable>:8008`
+
+:::note NOTE
+Make sure to rename `<element-vegetable>` with your actual robot name.
+:::
+
+This app runs persistently in the background, even when not active on the robot's display,
+ensuring that you can control your Amiga from any web browser at any time.
```

Comment:
```suggestion
You can control your Amiga from a web browser at any time, even when the Autoplot app is not active on the Brain display.
```

"app runs persistently in the background" is a bit of implementation detail, and might not be the best way to describe it in the future.